pkg/kubectl/plugins/plugins_test.go

```diff
@@ -55,6 +55,16 @@ func TestPlugin(t *testing.T) {
 			},
 			expectedErr: ErrInvalidPluginName,
 		},
+		{
+			plugin: &Plugin{
+				Description: Description{
+					Name:      "test",
+					Use:       "test spaces",
```

Comment:
This test can not test whether `GetUse` returns the right value. May add a new test for this.

Reply:
Added test. Thanks for reviewing.